### DISCUSSION TITLE
kpanda/user-guide/namespaces/exclusive/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1354] [Announcements]
# kpanda/user-guide/namespaces/exclusive/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/namespaces/exclusive/

**Comment by panpan0000** [comment]
那么在NS列表里，能看到它是否是一个被独享的NS么？

**Reply by lemontrees123** [reply]
可以在NS详情中看到独享了哪些节点，目前NS列表和节点上还无法看到
